connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java

```diff
@@ -131,7 +135,22 @@ public void connect(RemoteServer remoteServer) {
         ChunkUtils.sendEmptyChunks(this, playerEntity.getPosition().toInt(), 0, false);
 
         BiomeDefinitionListPacket biomePacket = new BiomeDefinitionListPacket();
-        biomePacket.setTag(CompoundTag.EMPTY);
+        InputStream stream = GeyserConnector.class.getClassLoader().getResourceAsStream("bedrock/biome_definitions.dat");
```

Comment:
This should be done in `connector/utils/Toolbox.java`, similar to how `runtime_block_states.dat` is handled.
Currently you are loading the `biome_definitions.dat` file every time a player connects.

Reply:
Fixed it! 😄

Reply:
👍